static/dotbotsim.js

```diff
@@ -152,53 +152,10 @@ function drawDotBots(data) {
             .attr("cx", function(d) { return scaleFactor*d.x; })
             .attr("cy", function(d) { return scaleFactor*d.y; })
             .attr("class", "dotBot")
-            .attr("fill",function(d,i){return dotBotcolors[i%10];})
+            .attr("fill",function(d,i) {return d.relay === true ? 'yellow' : dotBotcolors[i%10];})
```

Comment:
you are having the error you describe because you put this line only in `enter()`: this code will only execute when a new mote is added to the list, i.e. only when the page loads first (or when refreshing the page, which results in a re-load of everything). Copy this exact line also in the `transition()` portion after line149. Make sure there are no errors in your browser's console.

Reply:
I just tried that but the UI doesn't open at all and gives the error : Uncaught ReferenceError: gettingThingsInPlace is not defined. but nothing else changed.